templates/setenv.sh.erb

```diff
@@ -88,7 +88,11 @@ CATALINA_OPTS="-XX:+IgnoreUnrecognizedVMOptions ${CATALINA_OPTS}"
 CATALINA_OPTS="-Xlog:gc+age=debug:file=$LOGBASEABS/logs/gc-`date +%F_%H-%M-%S`.log::filecount=5,filesize=2M ${CATALINA_OPTS}"
 <%-     end -%>
 <%-   end -%>
+<%- if @jvm_type == 'openjdk-11' -%>
```

Comment:
Since this introduces conditional logic, it should have some simple spec tests that show that a string (line in file) exists based on the jvm_type.

Reply:
Added tests and spotted implementation errors (which i then fixed) by doing that, thanks for the hint!